modules/kamon/src/main/resources/kamon.conf

```diff
@@ -0,0 +1,43 @@
+kamon {
```

Comment:
rename the file to `reference.conf` such that the configuration is picked up automatically when adding this module as a dependency.